modules/scripts/startup-script/README.md

```diff
@@ -162,6 +162,24 @@ sudo journalctl -u google-startup-scripts.service
   use: [homefs, startup]
 ```
 
+```yaml
+- id: startup
+  source: ./modules/scripts/startup-script
+  settings:
+    gcs_bucket_full_path: gs://user-test-bucket/folder1/folder2
```

Comment:
I think we will want to add or update a test to use this. I can help with this next week.

Reply:
I am creating a task for this to be done in a follow on PR. It is something that will be picked up by the HPC Toolkit team unless you (@soumyapani) explicitly want to take it on.